### ISSUE TITLE
Hekili frames intermittently going invisible in Mythic Uunat

### ISSUE THREAD

**Issue by JMox** [Hekili/hekili#139]
Been using hekili for years, first issue I've had :( 
The frames (Primary rotation icons) go invisible periodicly during Mythic Uunat fight mostly when the Undying Guardians spawn in the fight but not always, it's really temperamental. 

Doesn't occur anywhere else other than this fight.

I've put the information from the addon below.
```
build: v8.1.5-17
level: 120
class: ROGUE
spec: assassination

talents: cheat_death
    elaborate_planning
    iron_wire
    master_assassin
    poison_bomb
    toxic_blade
    vigor

traits: azerite_empowered=3
    blightborne_infusion=1
    double_dose=3
    elemental_whirl=1
    footpad=1
    nothing_personal=2
    overwhelming_power=2
    resounding_protection=2

sets/legendaries/artifacts: none

gear:     sea_dogs_cuffs=1
    loa_exultants_shroud=1
    seal_of_the_zandalari_empire=1
    gorak_tuls_mantle=1
    mechjockey_grips=1
    blight_toadskin_leggings=1
    cincture_of_the_azerite_arsenal=1
    spymasters_wrap=1
    heart_of_azeroth=1
    tidemothers_cover=1
    stormglide_steps=1
    lurkers_insidious_gift=1
    variable_intensity_gigavolt_oscillating_reactor=1
    footbomb_championship_ring=1
    greeds_folly=1
```
```
Message: Interface\AddOns\Hekili\State.lua:3635: C stack overflow
Time: Thu May 16 19:20:14 2019
Count: 5675
Stack: Interface\AddOns\Hekili\State.lua:3635: C stack overflow
[C]: ?
Interface\AddOns\Hekili\State.lua:3635: in function <Interface\AddOns\Hekili\State.lua:3578>
Interface\AddOns\Hekili\State.lua:3625: in function <Interface\AddOns\Hekili\State.lua:3578>
...terface\AddOns\Hekili\Classes\RogueAssassination.lua:566: in function <...terface\AddOns\Hekili\Classes\RogueAssassination.lua:566>
(tail call): ?
...terface\AddOns\Hekili\Classes\RogueAssassination.lua:567: in function <...terface\AddOns\Hekili\Classes\RogueAssassination.lua:567>
(tail call): ?
Interface\AddOns\Hekili\State.lua:1690: in function <Interface\AddOns\Hekili\State.lua:1381>
Interface\AddOns\Hekili\State.lua:3144: in function <Interface\AddOns\Hekili\State.lua:3142>
Interface\AddOns\Hekili\State.lua:2074: in function <Interface\AddOns\Hekili\State.lua:2069>
...terface\AddOns\Hekili\Classes\RogueAssassination.lua:566: in function <...terface\AddOns\Hekili\Classes\RogueAssassination.lua:566>
(tail call): ?
...
Interface\AddOns\Hekili\State.lua:1690: in function <Interface\AddOns\Hekili\State.lua:1381>
Interface\AddOns\Hekili\State.lua:3144: in function <Interface\AddOns\Hekili\State.lua:3142>
Interface\AddOns\Hekili\State.lua:2074: in function <Interface\AddOns\Hekili\State.lua:2069>
...terface\AddOns\Hekili\Classes\RogueAssassination.lua:629: in function <...terface\AddOns\Hekili\Classes\RogueAssassination.lua:629>
(tail call): ?
...terface\AddOns\Hekili\Classes\RogueAssassination.lua:87: in function `last'
Interface\AddOns\Hekili\State.lua:1087: in function <Interface\AddOns\Hekili\State.lua:1046>
Interface\AddOns\Hekili\State.lua:4939: in function `reset'
Interface\AddOns\Hekili\Core.lua:1066: in function `ProcessHooks'
Interface\AddOns\Hekili\UI.lua:820: in function <Interface\AddOns\Hekili\UI.lua:684>

Locals: 
```

**Comment by Hekili** [comment]
Version 18-beta1 should be posted in a few minutes, please see if you still have this issue after updating.

Duplicate of #138

**Comment by Hekili** [comment]
Alright, I have a potential solution in v8.1.5-18-beta2 going up. Please let me know if it works for you!

**Comment by JMox** [comment]
@Hekili Haven't had an issue since updating, seems to be fixed in v8.1.5-18-beta2.
